scipy/interpolate/rbf.py

```diff
@@ -188,20 +185,18 @@ def _init_function(self, r):
         return a0
 
     def __init__(self, *args, **kwargs):
-        self.xi = np.asarray([np.asarray(a, dtype=np.float_).flatten()
-                           for a in args[:-1]])
+        self.xi = np.asarray([np.asarray(a, dtype=np.float_).flatten() for a in args[:-1]])
```

Comment:
Revert? The only change I see is taking a PEP8-compliant line and making it non-PEP8 compliant.